backends/magma/ceed-magma.h

```diff
@@ -22,13 +22,51 @@
 #define USE_MAGMA_BATCH
 #define USE_MAGMA_BATCH2
 #define USE_MAGMA_BATCH3
+#define USE_MAGMA_BATCH4
 
 void magma_dtensor_contract(Ceed ceed,
                             CeedInt A, CeedInt B, CeedInt C, CeedInt J,
                             const CeedScalar *t, CeedTransposeMode tmode,
                             const CeedInt Add,
                             const CeedScalar *u, CeedScalar *v);
 
+static int magma_dbasisApply_batched(
+        CeedBasis basis, CeedInt nelem, 
+        CeedTransposeMode tmode, CeedEvalMode emode,
+        const CeedScalar *u, CeedInt ustride, 
+              CeedScalar *v, CeedInt vstride) ;
+
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
Would `CEED_INTERN` work here?

Reply:
I haven't tried. I have significantly reduced the number of magma-related public symbols by using "static" kernels and device functions.